core/lib/src/error.rs

```diff
@@ -106,11 +115,14 @@ impl LaunchError {
     /// # Example
     ///
     /// ```rust
+    /// use rocket::error::Error;
     /// # if false {
-    /// let error = rocket::ignite().launch();
-    ///
-    /// // This line is only reached if launch failed.
-    /// let error_kind = error.kind();
+    /// if let Err(error) = rocket::ignite().launch() {
+    ///     match error {
+    ///         Error::Launch(err_kind) => println!("Found a launch error: {}", err_kind),
```

Comment:
This isn't quite right for a doc test for `kind()`. I think we should use `::Launch(err) => println!("Found a launch error: {}", err.kind())`